contents/julia_basics.md

```diff
@@ -2350,7 +2351,7 @@ By default, it returns a string that holds the file path for the downloaded file
 s = """
     url = "https://raw.githubusercontent.com/JuliaDataScience/JuliaDataScience/main/Project.toml"
 
-    my_file = Downloads.download(url) # tempfile() being created
+    my_file = download(url) # tempfile() being created
```

Comment:
```suggestion
    my_file = Downloads.download(url) # tempfile() being created
```

We need this because of conflicts with `Base.download`. See the failing GH actions...

Reply:
But when I do (in Windows using Julia v1.7)

```julia
using Downloads: download
url = "https://raw.githubusercontent.com/JuliaDataScience/JuliaDataScience/main/Project.toml"
my_file = Downloads.download(url)
```

I get `ERROR: LoadError: UndefVarError: Downloads not defined`. 

It seems to work if I just use

```julia
using Downloads
```

Reply:
This should work:
```
using Downloads: download
url = "https://raw.githubusercontent.com/JuliaDataScience/JuliaDataScience/main/Project.toml"
my_file = download(url)
```
Further discussion about `using Downloads` versus `using Downloads: download` is in the notation section (https://juliadatascience.io/notation). Specifically, see `6.2.3.1 Loading of symbols`.

Reply:
@rikhuijzer, I agree. That was my first suggestion (as you can see above) which then was overruled by @storopoli. So it seems both of you have to finally discuss what is "right"/better ...

Reply:
@Mo-Gul sorry, you might be right here. But could you please make a PR with a not-failing GH Actions CI.

I think you'll need to change the https://github.com/JuliaDataScience/JuliaDataScience/blob/main/src/JDS.jl#L52 to:

```julia
using Downloads: download
```

Reply:
@storopoli, sorry, but I leave this for you. I don't want to mess around with that stuff.

I would (in the meantime) continue *reading* the book after some days of filing issues and preparing PRs ;)

Reply:
See https://github.com/JuliaDataScience/JuliaDataScience/pull/223.